app/controllers/open_ai_controller.rb

```diff
@@ -0,0 +1,50 @@
+class OpenAiController < ApplicationController
+  def show
+    client = OpenAI::Client.new
+
+    @actions = YAML.load(File.read(Rails.root.join("config/arrays/wiki", "actions.yml")))
+    @values = YAML.load(File.read(Rails.root.join("config/arrays/wiki", "values.yml")))
+
+    merged_array = @actions.merge(@values)
+
+    description = ""
+    merged_array.each do |item|
+      if item.is_a?(Array) && item[1].is_a?(Hash) && item[1]['en-US'] == params[:prompt]
+        description = item[1]['description']
+        break
+      end
+    end
+
+    system_prompt = "You are a teacher, trying your best to explain programming terms within the context of the Overwatch Workshop.
```

Comment:
Have you used something like https://github.com/spdustin/ChatGPT-AutoExpert/blob/main/System%20Prompts.md for inspiration?